binder_helpers.py

```diff
@@ -48,7 +48,8 @@ def fix_reserved_word(word, is_module=False):
     """
     if is_module:
         if word == 'logging':
-            word = 'logging_'  # Still deciding this
+            # word = 'logging_'  # Still deciding this
+            pass
     else:
```

Comment:
@birdland can you remove this comment -- it will break the Python build. Or you'll have to merge in the dxtera PR for allowing this to be configured in the `configs.py` file.